src/Xpress.jl

```diff
@@ -28,20 +28,64 @@ const libxprs = joinpath(
 include("Lib/Lib.jl")
 include("helper.jl")
 include("api.jl")
-include("license.jl")
```

Comment:
I don't like the idea of moving all this code to the root file.
A user/developer should not start their trip into express from this license code.